pkgs/tools/backup/fabs/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, fetchFromGitHub
+, perl
+, python3
+, sqlite
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "fabs";
+  version = "1.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "openafs-contrib";
+    repo = "fabs";
+    rev = "v${version}";
+    hash = "sha256-ejAcCwrOWGX0zsMw224f9GTWlozNYC0gU6LdTk0XqH0=";
+  };
+
+  nativeBuildInputs = [
+    perl
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    alembic
+    dateutil
```

Comment:
This is an alias to `python-dateutil` and using it breaks eval without aliases enabled.

Fix included in #225732.

Reply:
Thanks! How could I have detected that myself? Is the only way to check against the list of aliases manually? nixpkgs-review does not seem to disable aliases when it runs...